text.html

```diff
@@ -1,70 +1,84 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
 </head>
+
 <body>
     <main>
         <header class="header">
             <div class="logo"><a href="#">Jo</a></div>
             <div></div>
-            <div class="menu">
-                <nav class="nav">
-                    <div ></div>
-                    <div ></div>
-                    <div ></div>
-                </nav>
-                
-            </div>
+            <nav class="nav">
+                <div><a href="#">Portfolio</a></div>
+                <div><a href="#About">About</a></div>
+                <div><a href="#Contact">Contact</a></div>
+            </nav>
         </header>
         <section class="headline">
             <div></div>
             <div class="greet">
                 <h1>I'm Jonathan Glad to see you!</h1>
-            </div> 
+            </div>
```

Comment:
Kindly make sure that your website looks good on mobile version also, so please fix the font-style of this heading to match the Figma design 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/192989813-5db094bf-f8d5-45db-ae8d-7127c7948aff.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/192989854-a05344b3-6b81-4a01-bf25-7c8fc0e81c04.png)